ports/espressif/common-hal/wifi/Radio.c

```diff
@@ -139,14 +139,14 @@ void common_hal_wifi_radio_set_mac_address(wifi_radio_obj_t *self, const uint8_t
     esp_wifi_set_mac(ESP_IF_WIFI_STA, mac);
 }
 
-uint8_t common_hal_wifi_radio_get_tx_power(wifi_radio_obj_t *self) {
+float common_hal_wifi_radio_get_tx_power(wifi_radio_obj_t *self) {
     int8_t tx_power;
     esp_wifi_get_max_tx_power(&tx_power);
-    return tx_power / 4;
+    return tx_power / 4.0;
```

Comment:
```suggestion
    return tx_power / 4.0f;
```

Reply:
`MICROPY_FLOAT_CONST(4.0)` is even more technically correct, even though we don't correctly follow that style everywhere (yet).

Reply:
Similar to how `mp_float_t` is either `float` or `double`, `MICROPY_FLOAT_CONST` appends the `f` to the literal constant (or not) depending on the actual type of `mp_float_t`.